rubycritic.gemspec

```diff
@@ -25,6 +25,7 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency 'flog', '4.3.2'
   spec.add_runtime_dependency 'reek', '3.11'
   spec.add_runtime_dependency 'parser', '~> 2.3'
+  spec.add_runtime_dependency 'ruby_parser', '>= 3.8'
```

Comment:
Well, `ruby_parser` isn't following semantic versioning (I think) and I'd imagine this is the kind of gem that wouldn't ever introduce incompatible changes. But it can't hurt to be a bit strict on this...
Will change in a bit